apps/web/src/pages/Users/UserInformationPage/components/UserCandidatesTable/UserCandidatesTable.tsx

```diff
@@ -0,0 +1,261 @@
+            : commonMessages.notFound,
+        ),
+      {
+        id: "priority",
+        header: intl.formatMessage(adminMessages.category),
+        cell: () => priorityCell(user.priorityWeight, intl),
+      },
+    ),
+    columnHelper.accessor(
+      ({ status }) =>
+        intl.formatMessage(
+          status ? getPoolCandidateStatus(status) : commonMessages.notFound,
+        ),
+      {
+        id: "finalDecision",
+        header: intl.formatMessage(tableMessages.finalDecision),
+        cell: ({ row: { original: poolCandidate } }) =>
+          finalDecisionCell(
+            intl,
+            poolCandidate,
+            unpackMaybes(poolCandidate?.pool?.assessmentSteps),
+          ),
+        enableSorting: false,
+      },
+    ),
+    columnHelper.accessor(
+      ({ status }) =>
+        intl.formatMessage(
+          status ? getPoolCandidateStatus(status) : commonMessages.notFound,
+        ),
+      {
+        id: "jobPlacement",
+        header: intl.formatMessage(tableMessages.jobPlacement),
+        cell: ({ row: { original: poolCandidate } }) =>
+          jobPlacementDialogAccessor(
+            poolCandidate as FragmentType<typeof JobPlacementDialog_Fragment>,
+            departments,
+          ),
+        enableSorting: false,
+      },
+    ),
+    columnHelper.accessor(
+      (row) => getLocalizedName(row.placedDepartment?.name, intl, true),
+      {
+        id: "placedDepartment",
+        header: intl.formatMessage(tableMessages.placedDepartment),
+        enableColumnFilter: false,
+      },
+    ),
+    columnHelper.accessor(
+      (poolCandidate) =>
+        candidacyStatusAccessor(poolCandidate.suspendedAt, intl),
+      {
+        id: "candidacyStatus",
+        header: intl.formatMessage(tableMessages.candidacyStatus),
+      },
+    ),
+    columnHelper.accessor(({ notes }) => notes || "", {
+      id: "notes",
+      header: intl.formatMessage(adminMessages.notes),
+      sortingFn: normalizedText,
+      cell: ({ row: { original: poolCandidate } }) =>
+        notesCell(intl, user.firstName, user.lastName, poolCandidate.notes),
+    }),
+    columnHelper.accessor(
+      () =>
+        intl.formatMessage(
+          user.preferredLang
+            ? getLanguage(user.preferredLang)
+            : commonMessages.notFound,
+        ),
+      {
+        id: "preferredLang",
+        header: intl.formatMessage(
+          commonMessages.preferredCommunicationLanguage,
+        ),
+      },
+    ),
+    columnHelper.accessor(() => user.email, {
+      id: "email",
+      header: intl.formatMessage(commonMessages.email),
+      sortingFn: normalizedText,
+      cell: () => cells.email(user.email),
+    }),
+    columnHelper.accessor(
+      () =>
+        currentLocationAccessor(user.currentCity, user.currentProvince, intl),
+      {
+        id: "currentLocation",
+        header: intl.formatMessage(tableMessages.currentLocation),
+      },
+    ),
```

Comment:
Same withe removing these. This information already appears on this page so it seems to just be duplicating it unnecessarily.

https://github.com/GCTC-NTGC/gc-digital-talent/pull/10342#issuecomment-2105265420